float64 x      # x-coordinate in plane
float64 y      # y-coordinate in plane
float64 z      # Height of plane (will be fixed, but can be at different heights)
float64 theta  # Orientation about z-axis in plane